### DISCUSSION TITLE
Train dataset

### DISCUSSION THREAD

**Discussion by jianjiandandande** [STVIR/pysot#454] [Q&A]
Why is the amount of data different in different training processes.

![image](https://user-images.githubusercontent.com/15951818/96447259-3ae38f00-1244-11eb-8535-f6c030520dcb.png)

![image](https://user-images.githubusercontent.com/15951818/96447409-72ead200-1244-11eb-8de9-2945749c3a43.png)

**Comment by ZhiyuanChen** [comment]
Could you provied detailed information on the set-up of these two training?
